operations/services.md

```diff
@@ -873,24 +879,25 @@ Creates a new order with the specified products and items. Only positive charges
 | --- | --- | --- | --- |
 | `ProductId` | string | required | Unique identifier of the [Product](services.md#product) to be ordered. |
 | `Count` | number | optional | Count of products to be ordered, e.g. 10 in case of 10 beers. |
-| `UnitCost` | [Cost](services.md#cost) | optional | Unit cost of the product that overrides the cost defined in Mews. |
+| `UnitAmount` | [Amount](services.md#amount) | optional | Unit amount of the product that overrides the cost defined in Mews. |
```

Comment:
link is incorrect. amount-value vs amount-parameters. Just amount doesn't exist.
```suggestion
| `UnitAmount` | [Amount](services.md#amount-parameters) | optional | Unit amount of the product that overrides the cost defined in Mews. |
```